jsonnet/rules.libsonnet

```
@@ -72,6 +72,10 @@ function(params) {
           expr: 'sum(rate(container_cpu_usage_seconds_total{container="",pod!=""}[5m])) / sum(machine_cpu_cores)',
           record: 'cluster:container_cpu_usage:ratio',
         },
+        {
+          expr: 'sum(irate(container_network_receive_bytes_total{pod!=""}[5m])) BY (pod, interface) + on(pod, interface) group_left(network_name) pod_network_name_info',
```

Fixing the many-to-many issue requires topk(1, ...).
I suppose that we want to preserve the namespace label too (it's unlikely to happen but nothing prevents 2 pods in different namespaces to have the same name, e.g. statefulset pods).

```suggestion
          expr: |||
              sum by(namespace,pod, interface) (irate(container_network_receive_bytes_total{pod!=""}[5m]))
              +
              on(namespace,pod, interface) group_left(network_name) topk by(namespace,pod, interface) (1, pod_network_name_info)
          |||,
```